__test__/integration/loan_offer/runners/create.ts

```diff
@@ -0,0 +1,19 @@
+import { Dharma } from "../../../../src";
+
+import { LoanOffer } from "../../../../src/types";
+
+export async function testCreate(dharma: Dharma, params: any) {
+    describe("passing valid params", () => {
+        let loanOffer: LoanOffer;
+
+        beforeAll(async () => {
+            loanOffer = await LoanOffer.create<LoanOffer>(dharma, params);
+        });
+
+        test("not signed by the debtor", async () => {
+            const isSignedByDebtor = await loanOffer.isSignedByDebtor();
```

Comment:
this test should just test that loanOffer is an instanceOf LoanOffer

Reply:
that's just a redundant test though. typescript already knows this